### PR TITLE
Add remaining font-awesome glyphs

### PR DESCRIPTION
**Pull request by mikebronner**

### CODE REVIEW

**Comment by mikebronner** [comment]
Addresses part of issue #98

File: Monoisome/Monoisome.fea [hunk 1]
```diff
@@ -114,569 +114,567 @@ feature liga {
...
-  sub f a hyphen c h e v r o n hyphen u p by ;
-  sub f a hyphen c h e v r o n hyphen d o w n by ;
-  sub f a hyphen r e t w e e t by ;
-  sub f a hyphen s h o p p i n g hyphen c a r t by ;
-  sub f a hyphen f o l d e r by ;
-  sub f a hyphen f o l d e r hyphen o p e n by ;
-  sub f a hyphen a r r o w s hyphen v by ;
-  sub f a hyphen a r r o w s hyphen h by ;
-  sub f a hyphen b a r hyphen c h a r t hyphen o by ;
-  sub f a hyphen b a r hyphen c h a r t by ;
```
Comment (larsenwork) [inline Monoisome/Monoisome.fea]: this seems gone?

**Comment by larsenwork** [comment]
Great work - only thing is it looks like you've accidentally deleted two lines?

File: Monoisome/Monoisome.fea (hunk 1 above)
Comment (mikebronner) [inline Monoisome/Monoisome.fea]: I may have accidentally deleted it thinking it was duplicated. Do you see others missing as well? I deleted a few that I thought were duplicate rows.
I will check any missing lines you find first thing in the morning (PST time). Sorry if that caused trouble.

Comment (larsenwork) [inline Monoisome/Monoisome.fea]: Not too much trouble - some icons just have multiple aliases (for some reason unknown to me) 

e.g. bar chart http://fontawesome.io/icon/bar-chart/ (small text to the right)

File: Monoisome/Monoisome.fea [hunk 2]
```diff
@@ -114,569 +114,567 @@ feature liga {
...
-  sub f a hyphen f i l e hyphen s o u n d hyphen o by ;
-  sub f a hyphen f i l e hyphen a u d i o hyphen o by ;
-  sub f a hyphen f i l e hyphen m o v i e hyphen o by ;
-  sub f a hyphen f i l e hyphen v i d e o hyphen o by ;
-  sub f a hyphen f i l e hyphen c o d e hyphen o by ;
-  sub f a hyphen v i n e by ;
-  sub f a hyphen c o d e p e n by ;
-  sub f a hyphen j s f i d d l e by ;
-  sub f a hyphen l i f e hyphen b o u y by ;
-  sub f a hyphen l i f e hyphen b u o y by ;
```
Comment (larsenwork) [inline Monoisome/Monoisome.fea]: this is the second one. All's good now :smile:

**Pull request merged**